src/elm/Session/Guest.elm

```diff
@@ -71,9 +71,13 @@ subscriptions model =
 -- VIEW
 
 
+{-| Page types the Guest can access.
+-}
 type Page
-    = Other
+    = Register
+    | Login
     | Shop
+    | Other
```

Comment:
I added Register and Login here to distinguish some styles for those pages for guest.